client/lxd_instances.go

```diff
@@ -210,6 +294,40 @@ func (r *ProtocolLXD) GetInstancesFullAllProjects(instanceType api.InstanceType)
 	return instances, nil
 }
 
+// GetFilteredInstancesFullAllProjects returns a filtered list of instances including snapshots, backups and state from all projects.
+func (r *ProtocolLXD) GetFilteredInstancesFullAllProjects(instanceType api.InstanceType, filters []string) ([]api.InstanceFull, error) {
```

Comment:
Maybe `GetInstancesFullAllProjectsWithFilter` instead (this is getting long ;))?